tools/generate_synth_data.py

```diff
@@ -0,0 +1,391 @@
+import click
```

Comment:
why should we sell this as part of our package?

i am not convinced that this tool makes a lot of sense as part of our lib — it's a cool inhouse tidbit, but i'd either explicitly define it in a test package or just not include it

Reply:
I would like to use it to create a quickstart that can be run directly from the repo without needing to download any additional resources. If we don't want to do that, then we can remove the script.